src/fuzz/fuzz-unit-file.c

```diff
@@ -48,7 +57,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
 
                 ll = l + strspn(l, WHITESPACE);
 
-                if (startswith(ll, "ListenNetlink=")) {
+                if (MSAN && startswith(ll, "ListenNetlink=")) {
```

Comment:
Would it make sense to put a similar check in `cg_slice_to_path`, where [`FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION`](https://github.com/systemd/systemd/blob/1238250f1dca07b8fcef4d7ebe86891334819e9e/src/basic/cgroup-util.c#L1979) is currently used to detect whether the code is run under `msan`?

Reply:
Done.